Source/WebCore/platform/Supplementable.h

```diff
@@ -121,7 +122,7 @@ class Supplementable {
 #endif
 
 private:
-    typedef HashMap<const char*, std::unique_ptr<Supplement<T>>, PtrHash<const char*>> SupplementMap;
```

Comment:
Note how the previous code was using `PtrHash` here. I'm kept pre-existing behavior except that the const char* is now wrapped in an ASCIILiteral.